dvc/command/metrics.py

```diff
@@ -210,7 +207,7 @@ def add_parser(subparsers, parent_parser):
         "--targets",
         nargs="*",
         help=(
-            "Metric files or directories (see -R) to show diff for. "
```

Comment:
Actually, in this case `-R` works, so we should keep it :slightly_smiling_face:  But probably need to rephrase it a bit, because it is about directories containing metric files.

Reply:
Updated the message. May still be badly phrased though

Reply:
I think the new text is accurate (taken from older commands like push/pull). Should we also apply it to metrics show though? I think so!

Reply:
Changed text in metrics show. Please, check it out.